app/canvas/info.h

```diff
@@ -8,8 +8,8 @@ class Datum;
 
 struct CanvasInfo
 {
-    QMap<Node*, QPointF> inspector;
-    QMap<Datum*, QPointF> subdatum;
+    QMultiMap<Node*, QPointF> inspector;
+    QMultiMap<Datum*, QPointF> subdatum;
```

Comment:
I don't understand these changes – [`QMap`](https://doc.qt.io/qt-6/qmap.html) is still supported, as is [`operator[]`](https://doc.qt.io/qt-6/qmap.html#operator-5b-5d).

Reply:
QMap is still supported but the [unite](https://doc.qt.io/qt-5/qmap-obsolete.html#unite) method (used [here](https://github.com/mkeeter/antimony/blob/ee525bbdad34ae94879fd055821f92bcef74e83f/app/canvas/info.cpp#L5)) has been deprecated and moved to this QMultiMap subclass, that handle multiple values per-key. The `[]` operator has been made private as they want you to use `value()` or `replace()` instead.